crates/crates-io/lib.rs

```diff
@@ -545,7 +545,7 @@ pub fn check_token(token: &str) -> Result<()> {
         Ok(())
     } else {
         Err(Error::InvalidToken(
-            "token contains invalid characters.\nOnly printable ISO-8859-1 characters \
+            "token contains invalid characters.\nOnly printable ASCII characters \
```

Comment:
Looking at #11600, it seems that what was checked evolved but the error message was never updated for the subset encoding that more people will also recognize.

So this seems reasonable